### PR TITLE
feat: add a talk about Open Source, forum PHP 2020

### PR DESCRIPTION
**Pull request by mere-teresa**
Add a talk given by me during the Forum PHP about Open source

### CODE REVIEW

**Review by Slashgear — APPROVED** [review]

**Review by pmartin — APPROVED** [review]

**Review by Oliboy50 — APPROVED** [review]

**Review by Oliboy50 — COMMENTED** [review]
~I can't see your article in https://pr-341.dsvmt7xpjktgx.amplifyapp.com/ nor your video in https://pr-341.dsvmt7xpjktgx.amplifyapp.com/meetups/ 🤔~

EDIT: my bad, I'm blind 🙈

File: _posts/2020-10-23-l'open-source-ce-n-est-pas-que-pour-le-web.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: video
+youtubeId: riPzJ-IMs6k 
+title: L'open source, ce n'est pas que pour le web - Forum PHP 2020
+description:  un tour d’horizon des initiatives open source dans d’autres domaines que l’informatique, pour en apprendre un peu plus sur la culture du libre.
+author: sarah-haim
```
[the remark this answers is not in the document]
Comment (Oliboy50) [reply _posts/2020-10-23-l'open-source-ce-n-est-pas-que-pour-le-web.md:6]: that's what I'm talking about: https://github.com/BedrockStreaming/tech.bedrockstreaming.com/pull/340/files#diff-fdca1c48b249c38bee14489176341e078b9cfcf168e19ba1c669b92fa7322c49R59-R62

but I don't know what's the impact of adding these author data

[the remark this answers is not in the document]
Comment (mere-teresa) [reply _posts/2020-10-23-l'open-source-ce-n-est-pas-que-pour-le-web.md:6]: Should I add my name AND a picture in the YAML file as an author?

**Comment by mere-teresa** [comment]
Added author data in YAML file.

**Pull request merged**